CHANGELOG.md

```diff
@@ -15,6 +15,12 @@
 
   The CLI now provides the subcommand `project migrate` which migrates Bolt projects to the latest version. When migrating a project, the [inventory file](https://puppet.com/docs/bolt/latest/inventory_file.html) will be changed from `v1` to `v2`. Changes are made in place and will not preserve comments or formatting.
 
+## Bug fixes
+
+* **Display all target aliases in `bolt inventory show --detail`** ([#1379](https://github.com/puppetlabs/bolt/issues/1379))
+
+  Bolt will now display aliases from all groups that a target is a member of in the output for `bolt inventory show --detail`. Previously, only the rightmost alias was shown in the output.
```

Comment:
Bolt now displays aliases from all groups, where a target is a member, in the output for `bolt inventory show --detail`. Previously, only the rightmost alias appeared in the output.
```suggestion
  Bolt will now display aliases from all groups that a target is a member of in the output for `bolt inventory show --detail`. Previously, only the rightmost alias was shown in the output.
```